bor/beginblocker.go

```diff
@@ -1,6 +1,8 @@
 package bor
 
 import (
+	"encoding/json"
```

Comment:
Could you use the same json package as we are already using - go-iter?

Reply:
Yes, I will make the change